src/AppServer.js

```diff
@@ -143,11 +144,14 @@ export class AppServer {
   async start ({port = this.port} = {}) {
     this.port = await getPort({port})
     await this.httpServer.listenAsync(this.port)
-    console.log(`
+    console.log(chalk`
     ===============================================================
-      Appstrap loaded successfully.
+      {yellow.bold Appstrap} loaded {green successfully}.
       A server has been started for you at the following address: 
-      http://localhost:${this.port}
+      {blue http://localhost:${this.port}}
+      
+      The management interface is located at the following address: 
```

Comment:
the management interface does not load in all circumstances.  We only start the management interface when invokedFromCLI is true.  This saves overhead from spinning up the vhost when we are in headless mode (for unit tests)

We'll need to add a condition to only add this part when invokedFromCLI is true.  If that means we need to assign that value to `this` for the appstrap class, feel free to do so.

Reply:
Come to think of it - we should really only be showing the entire message if invokedFromCLI is true.  This will cut down on the unnecessary console.log spam in peoples' tests as well.